### PR TITLE
Featuree/detail page mobile design

### PR DESCRIPTION
**Pull request by itsnameissteven**
## What does this PR do?
* Test added to the cypress spec.
* Mobile design has been added to the detailed information page
* Add a fixture to test data
-

## How should it be tested in the UI?
test for responsive nice
run cypress to make sure all test are passing

-

### CODE REVIEW

File: cypress/integration/metAPI-testing.ts [hunk 1]
```diff
@@ -35,4 +35,124 @@ describe('metAPI', () => {
...
+    cy.fixture('apiData').then(( data ) => {
+      cy.intercept('https://api.publicapis.org/entries', {
+        statusCode: 200,
+        body: data
+      })
+    })
+    cy.visit('http://localhost:3000/')
+  })
+
+  it('Should be able to click on an api to see more information', () => {
```
Comment (Jacksonmcguire) [inline cypress/integration/metAPI-testing.ts:51]: This is awesome, thanks for getting all of these tests up, much appreciated!

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -128,6 +133,7 @@ class App extends React.Component<Props> {
                         toggleFavorite={this.toggleFavorite} 
                         saveNote={this.saveNote} 
                         favorites={this.state.favorites} 
+                        resetHome={() => this.setState({ currentList: this.state.apiList })}
```
Comment (Jacksonmcguire) [inline src/components/App/App.tsx]: Definitely could be wrong, but I think currentApis is the state that we've gotta reset, cant find currentList being used anywhere else

**Review by Jacksonmcguire — APPROVED** [review]
All looks good, one minor thing is just the currentList setstate which I left a comment about. Great stuff with the testing, its looking super robust now.

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -58,18 +58,22 @@ class App extends React.Component<Props> {
...
+    if(!isSaved){
       this.setState({ favorites: [...this.state.favorites, ApiCard] });
     } else {
       const updatedFavorites = this.state.favorites.filter(element => element.API !== ApiCard.API)
       this.setState({ favorites: updatedFavorites})
     }
+    setTimeout(() => {
+      !this.state.favorites.length && localStorage.setItem('favorites', JSON.stringify(this.state.favorites));
+    },0)
   };
```
Comment (cbmackintosh) [inline src/components/App/App.tsx:72]: What does this setTimeout function do? Just interested

**Review by cbmackintosh — APPROVED** [review]
Tested screen responsiveness on my browser and ran through all the Cypress tests on my end. Everything looks good. Thanks for working through all those tests, much appreciated!

File: src/components/App/App.tsx (hunk 3 above)
Comment (itsnameissteven) [reply src/components/App/App.tsx:72]: It forces this statement into a queue so that the state has time to finish updating before this runs.

**Pull request merged**